firmware/user/display/sprite.c

```diff
@@ -13,6 +13,14 @@
  */
 int16_t ICACHE_FLASH_ATTR plotSprite(int16_t x, int16_t y, const sprite_t* p_sprite, color col)
 {
+    // Only attempt to plot the sprite if some of it will be in bounds
+    if(x >= OLED_WIDTH || x <= -p_sprite->width || y >= OLED_HEIGHT || y <= -p_sprite->height)
```

Comment:
As Richard pointed out a while ago, this will segfault because `width` and `height` are 8 bit fields and data in ROM must be accessed at 32 bit boundaries.

At this point, it's safe to say these functions will only be used for text, which is why I hardcoded 12 in the example I gave you.